swap/src/lib.rs

```diff
@@ -63,3 +66,10 @@ impl Display for SwapAmounts {
         )
     }
 }
+
+#[derive(Debug, Clone, Copy)]
+pub enum ExpiredTimelocks {
+    None,
+    Cancel,
+    Punish,
+}
```

Comment:
This belongs in the `bitcoin` module. We want to remove `lib.rs` so let's not add more to it.